### PR TITLE
Update for django 1.8

### PR DESCRIPTION
**Pull request by meshy**
There's a failing test related to testing the fields on Route.
In django 1.8 we seem to be getting the related names of models that were created in other test methods.

### CODE REVIEW

**Comment by KevinEtienne** [comment]
I was thinking that [Django](https://github.com/django/django/blob/aaacaeb0963c406c4fe6f68c6ae51f4a65878250/django/db/models/options.py) might have a method to query the non-related fields and related fields separately which might have helped here but I didn't find anything relevant here.

**Comment by meshy** [comment]
I think that [`._meta.get_all_field_names()` is deprecated](https://docs.djangoproject.com/en/1.8/ref/models/meta/#migrating-old-meta-api), so we should perhaps replace it with something in `incuna-test-utils`.

**Comment by meshy** [comment]
That said, I can't help but question the use of the test now.

**Comment by Ian-Foote** [comment]
Perhaps [`field_names`](https://github.com/incuna/incuna-test-utils/blob/master/incuna_test_utils/utils.py#L33-L39) is appropriate here?

**Comment by meshy** [comment]
Sounds like a reasonable plan

**Comment by meshy** [comment]
Alternatively, is there a better way to create the subclasses we're testing?

**Comment by meshy** [comment]
@Ian-Foote `field_names` returned a very similar list of fields, and did include the two that are making the tests fail.

**Comment by Ian-Foote** [comment]
Ah, that's a pity.

**Comment by meshy** [comment]
I added the new fields to what the test expected.

File: conman/routes/migrations/0001_initial.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    dependencies = [
+        ('contenttypes', '0002_remove_content_type_name'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Route',
+            fields=[
+                ('id', models.AutoField(auto_created=True, verbose_name='ID', primary_key=True, serialize=False)),
+                ('slug', models.SlugField(help_text='\n        Used to create the location of the Route. The Root Route needs\n        "slug" to be blank; all other Routes need a value unique to the parent.\n        It can only contain letters, numbers, underscores, or hyphens.\n    ', default='', max_length=255)),
```
Comment (Ian-Foote) [inline conman/routes/migrations/0001_initial.py]: I'm not sure the [`slug.help_text`](https://github.com/meshy/django-conman/blob/master/conman/routes/models.py#L55-L59) is correctly defined. Perhaps it should be using [`textwrap.dedent`](https://docs.python.org/3.4/library/textwrap.html#textwrap.dedent)?

Comment (meshy) [inline conman/routes/migrations/0001_initial.py]: Yeah, I think you're right, this is odd. Really, it should just be one line, as that's how I am expecting it to appear in the admin.

https://github.com/meshy/django-conman/issues/48

**Comment by meshy** [comment]
\o/

**Pull request merged**